search-service/src/main/kotlin/com/egm/stellio/search/util/QueryUtils.kt

```diff
@@ -14,20 +14,24 @@ import java.util.Optional
 fun parseAndCheckQueryParams(
     pagination: ApplicationProperties.Pagination,
     requestParams: MultiValueMap<String, String>,
-    contextLink: String
+    contextLink: String,
+    pointOfTimeIsNeeded: Boolean = false
```

Comment:
not easy to understand what it is about. better go with something that reminds the terms of the specification, for instance `temporalQueryRequired`.